test/application_test.rb

```diff
@@ -6,8 +6,8 @@
   end
 
   it 'instantiate new configuration object when inherited' do
-    backend_app  = Backend::Application.new
-    frontend_app = Frontend::Application.new
+    backend_app  = Back::Application.new
```

Comment:
That would be better, Although I think this issue to be solved in another PR. I'll rollback this change from this PR.